Python/Face_Detection/face-detection.py

```diff
@@ -0,0 +1,44 @@
+#Face and Eye Detection 
```

Comment:
@anubhav201241 Am sorry I cant validate this PR. The script shares uncanny similarities with [this](https://pythonprogramming.net/haar-cascade-face-eye-detection-python-opencv-tutorial/). Am not claiming that the scripts in the PR a straight cope paste, but other than renaming the variables (although naming paths as CONSTANT is a good idea) and mirroring the rectangle box, nothing has been changed.

Even if the code is not plagiarised, it exists [here](https://github.com/farcompen/Face-Eye-Detection/blob/master/goz_takip_image.py), [here](https://github.com/akshaybahadur21/FaceDetection/blob/master/Face.py), [here](https://datascienceplus.com/face-and-eye-detection-with-opencv/) and [here](https://gist.github.com/icyfiremoin/22571df6e4bdd4f569cd7532d06c10e0).

Also [here](https://www.blog.pythonlibrary.org/2018/08/15/face-detection-using-python-and-opencv/), [_which interestingly enough should never work_].

If the code already exists at so many sources, what's the point of having it here. If the script would have explained the steps of detection like [this](https://github.com/informramiz/opencv-face-recognition-python/blob/master/OpenCV-Face-Recognition-Python.py).

It would have been my pleasure to merge this ASAP.